recipes/olc-pge/all/conanfile.py

```diff
@@ -0,0 +1,70 @@
+from conans import ConanFile, tools
+
+
+class oldPixelGameEngineConan(ConanFile):
+    name = "olc-pge"
+    homepage = "https://github.com/OneLoneCoder/olcPixelGameEngine"
+    description = "The olcPixelGameEngine is a single-file prototyping and game-engine framework created in C++. It is cross platform, compiling on Windows via Visual Studio and Code::Blocks, and on Linux with a modern g++."
+    topics = ("conan", "olc", "pge", "pixelengine", "pixelgameengine", "pgex",
+              "game-development", "game-engine", "engine", "gamedev", "gaming", "graphics")
+    license = "OLC-3"
```

Comment:
This is a custom license with no SPDX entry. Please use `LicenseRef-LICENSE` for such cases, to basically state "custom license, look in the LICENSE file"

```suggestion
    license = "LicenseRef-LICENSE"
```

Ref: https://github.com/conan-io/conan-center-index/pull/4928/files#r596208293

Reply:
Is it not BSD-3-Clause?

https://github.com/OneLoneCoder/olcPixelGameEngine/blob/master/LICENCE.md
https://opensource.org/licenses/BSD-3-Clause